### ISSUE TITLE
API Proposal: OpenTok.getStream() to get stream information

### ISSUE THREAD

**Issue by jeffswartz** [opentok/opentok-node#178]
This is a proposal to add a new OpenTok Node.js SDK method for getting stream info.

## `OpenTok.getStream()`

This new method gets info about a stream. The stream must be an active stream in
an OpenTok session. See the
[OpenTok REST API documentation](https://tokbox.com/developer/rest/#get-stream-info).

```javascript
# To get stream info:
var sessionId = '2_MX6xMDB-fjE1MzE3NjQ0MTM2NzZ-cHVTcUIra3JUa0kxUlhsVU55cTBYL0Y1flB';
var streamId = '2a84cd30-3a33-917f-9150-49e454e01572';
opentok.getStream(sessionId, streamId, function(error, streamInfo) {
  if (error) {
    console.log(error.message);
  } else {
    console.log(stream.id); // '2a84cd30-3a33-917f-9150-49e454e01572'
    console.log(stream.videoType); // 'camera'
    console.log(stream.name); // 'Bob'
    console.log(stream.layoutClassList); // ['main']
  }
}
```

**Comment by msach22** [comment]
@jeffswartz This looks good, but do you think the developer should be able to call the `getStream` method from the `Session` object (returned by `createSession`) as well? Something like this:

```
session.getStream(streamId, function(error, streamInfo) {
  if (error) {
    console.log(error.message);
  } else {
    console.log(stream.id); // '2a84cd30-3a33-917f-9150-49e454e01572'
    console.log(stream.videoType); // 'camera'
    console.log(stream.name); // 'Bob'
    console.log(stream.layoutClassList); // ['main']
  }
}
```

**Comment by jeffswartz** [comment]
@msach22 -- Some other session-related methods exist on the OpenTok object only. Should we also add these methods to the Session object:

* `Session.signal(connectionId, payload, callback)`
* `Session.forceDisconnect(connectionId, callback)`
* `Session.dial(token, sipUri, options, callback)`

**Comment by msach22** [comment]
Yes, I think we should be consistent with our message. Since `generateToken` is available through the `session` object, we should also provide `signal`, `forceDisconnect`, and `dial` through them.

**Comment by msach22** [comment]
This has been released with `2.7.0`
